.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
+
+[*]

```

Comment:
That wildcard is dangerous, we should have an exhaustive list of extensions.

Reply:
Removed